src/database/index.ts

```diff
@@ -594,6 +594,22 @@ export class Database {
     return application ?? null;
   }
 
+  async getApplicationByProjectId(
+    chainId: ChainId,
+    roundId: string,
+    projectId: string
+  ) {
+    const application = await this.#db
+      .selectFrom("applications")
+      .where("chainId", "=", chainId)
+      .where("roundId", "=", roundId)
+      .where("projectId", "=", projectId)
+      .selectAll()
+      .executeTakeFirst();
+
+    return application ?? null;
+  }
```

Comment:
^ Just a call out. This won't return the allo-v1 migrated applications given a allo-v2 id

Reply:
@thelostone-mc  is the migration removing the original application or adding a new application with the new id?

Reply:
migration simply stores a link btwn the old and new id.
we don't actually migrate the old application